cdap-api/src/main/java/co/cask/cdap/api/workflow/AbstractWorkflowAction.java

```diff
@@ -38,22 +40,18 @@ protected AbstractWorkflowAction(String name) {
     this.name = name;
   }
 
+  @Override
   public void configure(WorkflowActionConfigurer configurer) {
     this.configurer = configurer;
-    WorkflowActionSpecification specification = configure();
-    configurer.setName(specification.getName());
-    configurer.setDescription(specification.getDescription());
```

Comment:
There is no more `WorkflowActionSpecification configure()` method, hence there is nothing to copy from the spec.